### ISSUE TITLE
Timestamp with milliseond precision

### ISSUE THREAD

**Issue by trbck** [danni-m/redis-timeseries#26]
Hi, your (excellent) module does not seem to support millisecond precision as it throughs a "timestamp is too old" error. Any chance this might be added in the future?

**Comment by danni-m** [comment]
Yes, Keeping a sub-second samples are technically doable, it will take more space since we don't have any compression at the moment, and even with compression it will be less memory efficient.

I would suggest to try and compile the module after settings `timestamp_t` and `api_timestamp_t` to int64_t and see if it works.
